api/pow/client/client.go

```diff
@@ -0,0 +1,121 @@
+package client
+
+import (
+	"context"
+
+	ffsRpc "github.com/textileio/powergate/ffs/rpc"
+	healthRpc "github.com/textileio/powergate/health/rpc"
+	netRpc "github.com/textileio/powergate/net/rpc"
+	walletRpc "github.com/textileio/powergate/wallet/rpc"
+	"google.golang.org/grpc"
+)
+
+// Client provides the client api.
+type Client struct {
+	healthC healthRpc.RPCServiceClient
+	netC    netRpc.RPCServiceClient
+	ffsC    ffsRpc.RPCServiceClient
+	walletC walletRpc.RPCServiceClient
+	conn    *grpc.ClientConn
+}
```

Comment:
Ended up adding all the pow client code as it's own package. Didn't make sense (for `FFS` related code) to add it as a sub package of `users` or `hub` apis because `FFS` related calls are relevant to both `User`s and `Account`s.

Reply:
It's just one file/package now, we can always split it up if we add more apis in here later

Reply:
CC @andrewxhill similar additions will be made to the JS Hub client next.

Reply:
cool, makes sense

Reply:
well... wait, what do you mean "Users and Accounts"? 

there is no Accounts object, class, API used in JS

Reply:
Oh I was just talking about the structure of the client API in Go... It has a package `hub` that is used by developers to manage their dev and org `Account`s and a package `users` that is used by end `User`s to interact with threads. Powergate interactions make sense in both of those contexts (because FFS instances are created for both `User`s and `Account`s), so in this case, I just keep the powergate client code in it's own package that can be used however makes sense.

In the case of `js-hub`, there is currently no equivalent of `hub` admin APIs, so we may choose to integrate the powergate apis in differently. If we ever did add the admin apis to the JS lib, we'd be facing the same situation of powergate apis applying to both admin `Account`s and end `User`s.